### PR TITLE
Put task directly on queue

### PR DESCRIPTION
**Pull request by lreesby**
command: `cargo build --release --features neumann-node --features dev-queue`

### CODE REVIEW

**Review by justinzhou93 — APPROVED** [review]
ahh what was causing the flakey compilation? 

and mind putting some instructions in a wiki or the README?

**Review by justinzhou93 — CHANGES_REQUESTED** [review]
whoops, didn't mean to approve. had 2 questions in the review above. once those are good to go, then approval from me!

File: pallets/automation-time/src/lib.rs [hunk 1]
```diff
@@ -926,6 +931,30 @@ pub mod pallet {
...
+					None => {
+						let task_ids: BoundedVec<T::Hash, T::MaxTasksPerSlot> =
+							vec![task_id].try_into().unwrap();
+						<ScheduledTasks<T>>::insert(current_time_slot, task_ids);
+					},
+					Some(mut task_ids) => {
+						if let Err(_) = task_ids.try_push(task_id) {
+							Err(Error::<T>::TimeSlotFull)?
+						}
+						<ScheduledTasks<T>>::insert(current_time_slot, task_ids);
```
[the remark this answers is not in the document]
Comment (lreesby) [reply pallets/automation-time/src/lib.rs:948]: insertion to task map happens here: https://github.com/OAK-Foundation/OAK-blockchain/pull/137/files#diff-e6a13f7032a383e19b77d20043ca46041d94f5e49396812e2de1b2000c301e59R1012

[the remark this answers is not in the document]
Comment (justinzhou93) [reply pallets/automation-time/src/lib.rs]: ahh right, i always forget we split that up

**Review by justinzhou93 — APPROVED** [review]

**Review by justinzhou93 — APPROVED** [review]

**Review by rhuttman — APPROVED** [review]

**Pull request merged**